src/apps/common/submodules/numbers/numbers.js

```diff
@@ -69,13 +64,13 @@ define(function(require) {
 
 				var arrayNumbers = data.listAccounts.length ? data.listAccounts[0].usedNumbers : [];
```

Comment:
Maybe a function similar to `numbersPaintSpare` could be created for used numbers, to reduce repeating code.

Reply:
`used numbers` list is needed once on the first load

Reply:
You are right, it is true